requirements.txt

```diff
@@ -0,0 +1,7 @@
+# - Python Module Requirements
+
+# Required for building plClient's resource.dat assets
+https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/cp27/pycairo-1.18.2-cp27-cp27m-win32.whl; sys_platform == "win32" and python_version ~= "2.7"
```

Comment:
???

Reply:
The old (Py2.7) version of CairoSVG tries to import cairo, which fails.  This manually installs a pre-compiled version of the module, and only does so on Windows with Python 2.7.  If/when we deprecate using Py2.7, this can be removed.  It's necessary for our 2.7 Windows builds on AppVeyor (and presumably any clean system that doesn't have cairo installed through the old installer on which we used to depend through PyGTK).